examples/README.md

```diff
@@ -1,78 +1,125 @@
+# Trying EventStoreDb connector
 
-[Nest](https://github.com/kamilmysliwiec/nest) framework Eventstore [CQRS module](https://github.com/kamilmysliwiec/nest-cqrs) usage example.
+## Note on nodeJS version
+
+You'll find a compatible version you refer to in [this .nvmrc](../.nvmrc). You might want to use nvm direcly, and in this case, nvm will read direcly this configuration file and adjust the version.
+
+## Installation
+
+```
+yarn install
+```
+
+## Start
+
+You have to run a eventstore container with correct ports. You can put the ports you want, but it will work natively with this conf :
+
+```
+docker run --name esdb-node \
+    -it -p 20113:2113 -p 10113:1113 \
+    eventstore/eventstore:latest \
+    --insecure \
+    --run-projections=All \
+    --enable-atom-pub-over-http
+```
+
+Then, the REST API will start by running :
+
+```
+yarn start
+```
+
+## Usage
+
+#### running basic CQRS example with eventStore
+
+The example is based on [CQRS module](https://github.com/kamilmysliwiec/nest-cqrs) usage example of [Nest](https://github.com/kamilmysliwiec/nest) framework.
 
 # Story
+
 The hero fight the dragon in an epic fight.
 he kills the dragon, and a dragon can be killed only once.
 We need to keep only the last 5 move of the fight and delete them after 3 days for faery RGPD.
 
 When it's done the hero search and find an item.  
 He can find this special item only once until he drop hit.
 
+#### How to connect
 
-## Installation
+You can see how the import is done in the [EventStoreHeroesModule](./src/heroes/event-store-heroes.module.ts) :
 
-```
-yarn install
+```typescript
+    CqrsEventStoreModule.register(
+      eventStoreConnectionConfig,
+      eventStoreSubsystems,
+      eventBusConfig,
+    ),
 ```
 
-## Start
-```
-docker run -d -p 2113:2113 -p 1113:1113 -e EVENSTORE_RUN_PROJECTIONS=System --name geteventstore eventstore/eventstore
-yarn start // or yarn start:<cqrs | eventstore | subscription | write>
-```
+# Let's run it
+
+Once the API is running, you only have to run this REST request :
 
-## Usage 
 ```
-curl -XPUT localhost:3000/hero/hero-5421/kill -d'{
-  "dragonId" : "test3"
-}'
+curl -XPUT localhost:3000/hero/hero-200/kill -d'{ "dragonId" : "test3" }'
```

Comment:
hero-200 need to be a number (see +heroId in kill-dragon.handler) otherwise there is an hero-NaN in the logs and a hero-NaN stream